src/ng/parse.js

```diff
@@ -1687,6 +1687,25 @@ function getValueOf(value) {
 function $ParseProvider() {
   var cacheDefault = createMap();
   var cacheExpensive = createMap();
+  var forceExpensiveChecks = false;
+
+  /**
+   * @ngdoc method
+   * @name $parseProvider#expensiveChecks
+   * @description
+   * Force expensive checks on all parsed expressions. Defaults to `false`.
```

Comment:
I think a little explanation on what `false` means is in place (i.e. that will still apply expensive checks in some places or even what "expensive checks" include).